examples/ppo/ppo_lib.py

```diff
@@ -274,31 +296,39 @@ def train(
   log_frequency = 40
   checkpoint_frequency = 500
 
-
-  for s in range(loop_steps):
+  initial_params = get_initial_params(jax.random.PRNGKey(0), model)
+  state = create_train_state(initial_params, model, config,
+                             loop_steps * config.num_epochs)
+  del initial_params
+  state = checkpoints.restore_checkpoint(model_dir, state)
+  # number of train iterations done by each train_step
+  iterations = config.num_agents * config.actor_steps // config.batch_size
+  start_step = int(state.step) // config.num_epochs // iterations
```

Comment:
(It's a bit confusing having `start_step` mean something quite different from `state.step`.)

Reply:
Yeah I agree this is confusing but somewhat hard to solve. For performance reasons we need to do multiple optimization steps in a single train_step. This is because RL has really tiny models so the overhead of a single train_step call would be non-trivial.

Reply:
And it doesn't help that we have an inner loop over `config.num_epochs` for every "step" :-)

Reply:
Yeah I have to dig into that again at some point. I think it made sense from an RL point of view but I need to think about it and put it in a docstring

Reply:
SGTM

Reply:
SGTM